CBI.Jenkinsfile

```diff
@@ -74,8 +72,6 @@ spec:
 
     stage('Build Xtext BOM') {
       steps {
-        dir('.m2/repository/org/eclipse/xtext') { deleteDir() }
```

Comment:
what happended here?

Reply:
sure this will work for releases?

Reply:
Pretty sure. Within containers the directory will always be empty on startup.